bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/type/ChannelGroupType.java

```diff
@@ -39,17 +40,20 @@
      * @param description the human readable description for the according type
      *            (could be null or empty)
      *
+     * @param category the category of this channel group type, e.g. Temperature (could be null or empty)
+     *
      * @param channelDefinitions the channel definitions this channel group forms
      *            (could be null or empty)
      *
      * @throws IllegalArgumentException if the UID is null, or the label is null or empty
      */
     public ChannelGroupType(ChannelGroupTypeUID uid, boolean advanced, String label, String description,
-            List<ChannelDefinition> channelDefinitions) throws IllegalArgumentException {
+            final String category, List<ChannelDefinition> channelDefinitions) throws IllegalArgumentException {
```

Comment:
Afaics, he mentioned the field, not the variable. Just like the fields for advanced and channelDefinitions are final, while the method parameters are not.